internal/utils/templates/init_config.toml

```diff
@@ -119,6 +119,20 @@ template = "Your code is {{ `{{ .Code }}` }} ."
 # enabled = true
 # uri = "pg-functions://<database>/<schema>/<hook_name>"
 
+# Use a custom sms sender
+[auth.hook.send_sms]
+# enabled = true
+# We use host.docker.internal to connect to port 54321 on the host. See: https://docs.docker.com/desktop/networking/
+# uri = "http://host.docker.internal:54321/functions/v1/sms_sender"
```

Comment:
Since this hook is called from inside auth container, it should be possible to use either `kong` or `edge_runtime` network alias, ie. `http://kong:8000/functions/v1/sms_sender`

Reply:
Sure let's use `kong`  so it goes through the gateway still.

Reply:
Ah sigh. This would require an auth level change

Reply:
Opened a pr, but for reference do you happen to know which platforms?

Reply:
Linux was known to have problems https://stackoverflow.com/a/62431165 so we added the default workaround with `host-gateway`. However the workaround doesn't work for older versions of podman https://github.com/containers/podman/issues/14390.

But it appears that all these issues are resolved now so may be it's ok. 😄